### ISSUE TITLE
Where is the dwrd being calculated and what does it represents? 

### ISSUE THREAD

**Issue by Imtiaz08** [osqzss/gps-sdr-sim#318]
Hi there's a variable called Dwrd which stores an array of 60 elements... Can anybody tell me what does it represents please? And where is it being calculated? 

It is so confusing please anybody???

**Comment by shuwang1** [comment]
I guess you know the answer already.

**Comment by Imtiaz08** [comment]
> I guess you know the answer already.

Yes, I've learned but I still don't understand when we have subframes already then what is the purpose of DWORD generation? 

I have generated a signal in Matlab but when I try to see it in the GNSS SDR it says couldn't find a valid preamble... I even tried generating the signal using GPS-SDR-SIM and the result was the same...
